### PR TITLE
#18:Create and style for course component

### PR DESCRIPTION
**Pull request by ngtrongnguyen**
Ảnh chụp Màn hình 2023-06-09 lúc 14 01 37

### CODE REVIEW

File: src/stylesheets/components/_card-rating.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
+@use '../abstract/' as *;
+
+:root {
```
Comment (Lanng21) [inline src/stylesheets/components/_card-rating.scss:3]: why u style root here?

File: src/stylesheets/components/_card-rating.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    --star-size: 16px;
+    --star-color: #D9D9D9;
```
Comment (Lanng21) [inline src/stylesheets/components/_card-rating.scss:5]: can u move it to variable.scss ?

**Comment by Lanng21** [comment]
please update the screenshot again

**Review by haunguyen15-tt — APPROVED** [review]

**Comment by Lanng21** [comment]
resolve conflicts

**Review by Lanng21 — APPROVED** [review]

**Pull request merged**